sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/Rules/RuleDescriptionExtensions.cs

```diff
@@ -29,7 +29,7 @@ public static void ValidateDescriptionName(this RuleProperties description)
 #pragma warning restore CA2208 // Instantiate argument exceptions correctly
             }
 
-            string[] uriSchemeKeys = { "@", "?", "#" };
+            string[] uriSchemeKeys = ["@", "?", "#"];
```

Comment:
Should we consider hoisting this to a `static readonly`?

Reply:
I was thinking about that as well but wasn't sure how far I should go in a single PR